app/src/main/res/values/styles.xml

```diff
@@ -1,6 +1,6 @@
 <resources>
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
```

Comment:
I'm glad that I didn't miss the perfectly balanced `Light.DarkActionBar`!  Jokes aside same question with that in https://github.com/Huy-Ngo/MiniGH/pull/12/files?short_path=de7e5ac#r543431912

Reply:
Our app does not have action/tool bar I suppose?

Reply:
wdym?

Reply:
I think he mean that at login, we have no action bar and at other activities (code, commit, etc), we have it.